src/utils/fetch.js

```diff
@@ -148,6 +148,20 @@ export function convertFetchSuccess(url, body, timerId, result) {
     return convertedPromise;
 }
 
+function getBody(method, options) {
+    if (!options) {
+        return null;
+    }
+
+    // If PATCH without body occurs, create empty payload.
+    // Reason: Some proxies and default configs for CDNs like Akamai have issues with accepting PATCH with content-length: 0.
+    if (method === 'PATCH' && !options.body && !options.useXHttpMethodOverride) {
+        return {};

```

Comment:
Empty object seems like safest option here. It's what openapi does in explorer. This also enforcers json format, which some endpoints expect. 

Will syncup with openapi if something might potentially break if we send just string with empty space.

However at this point this is the safest option.